homeassistant/components/tessie/button.py

```diff
@@ -67,11 +67,11 @@ class TessieButtonEntity(TessieEntity, ButtonEntity):
 
     def __init__(
         self,
-        coordinator: TessieStateUpdateCoordinator,
+        vehicles: TessieVehicleData,
```

Comment:
```suggestion
        vehicle: TessieVehicleData,
```
We should be consistent with the naming I think so it's easy to understand if it's one or many

Reply:
Good catch!